samples/tiled-convolution/CMakeLists.txt

```diff
@@ -1,4 +1,28 @@
 set(SOURCE_NAME "tiled-convolution")
+
+if(DEFINED TARGET)
```

Comment:
You're missing out on a case here: The user might have defined `TARGET` to something other than `RCAR`. In this case the `LOCAL_SIZE_x` variables are never set.

I don't think you need the surrounding check at all.

Reply:
Also, is `TARGET` used elsewhere already? If not, I'd prefer a more descriptive name. `TARGET` can mean a lot in build systems.